test/test_add_points_in_pointcloud.py

```diff
@@ -35,32 +40,42 @@ def setup_module(module):
 
 
 @pytest.mark.parametrize(
-    "epsg",
+    "input_file, epsg",
     [
-        "EPSG:2154",  # should work when providing an epsg value
-        None,  # Should also work with no epsg value (get from las file)
+        (INPUT_POINTS_2D, "EPSG:2154"),  # should work when providing an epsg value + GeoJSON POINTS 2D
+        (INPUT_POINTS_3D, "EPSG:2154"),  # should work when providing an epsg value + GeoJSON POINTS 3D
+        (INPUT_POINTS_2D, None),  # Should also work with no epsg value (get from las file) + GeoJSON POINTS 2D
+        (INPUT_POINTS_3D, None),  # Should also work with no epsg value (get from las file) + GeoJSON POINTS 3D
     ],
 )
-def test_clip_3d_points_to_tile(epsg):
-    # With EPSG
-    points_gdf = gpd.read_file(INPUT_POINTS)
-    points_clipped = add_points_in_pointcloud.clip_3d_points_to_tile(points_gdf, INPUT_PCD, epsg, 1000)
+def test_clip_3d_points_to_tile(input_file, epsg):
+    points_input = gpd.read_file(input_file)
+    points_clipped = add_points_in_pointcloud.clip_3d_points_to_tile(points_input, INPUT_PCD, epsg, 1000)
     assert len(points_clipped) == 678  # check the entity's number of points
 
 
 @pytest.mark.parametrize(
-    "epsg",
+    "input_file, epsg",
     [
```

Comment:
cf. commentaire dans les lignes suivantes

je pense que ce serait bien de rajouter un commentaire ici du type : 

```suggestion
    "input_file, epsg",
    # Test on the same geomtries contained in various geometry formats
    [
```